packages/components/src/components/checklist/checklist.tsx

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
+import React, { MouseEventHandler } from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import Icon from '../icon';
 import Button from '../button';
 
-const ItemStatus = ({ status, onClick, button_text }) => {
+type TItemStatusProps = {
+    button_text: string;
+    onClick: MouseEventHandler<HTMLButtonElement | HTMLDivElement>;
```

Comment:
as I see in all packages we don’t use separate imports for all events or hooks, maybe we can stay we the same: React. MouseEventHandler ?

Reply:
yes it would be better to keep it same as others. thanks